common-lib/common/panels/timeSeries/network/base.libsonnet

```diff
@@ -0,0 +1,26 @@
+local g = import '../../../g.libsonnet';
+local base = import '../base.libsonnet';
+local timeSeries = g.panel.timeSeries;
+local fieldOverride = g.panel.timeSeries.fieldOverride;
+local custom = timeSeries.fieldConfig.defaults.custom;
+local defaults = timeSeries.fieldConfig.defaults;
+local options = timeSeries.options;
+base {
+  new(
+    title,
+    targets,
+    description=''
+  ):
+    super.new(title, targets, description)
+    + self.standardOptions.withDecimals(1)
+    + self.standardOptions.withUnit('pps'),
+
+  withNegateOutPackets(regexp='/transmit|tx|out/'):
```

Comment:
Love this. It's such a visually expressive way to show network packets, and it's a lot of copy/paste or UI toil to set these up generally. Having it in a reusable library/pattern is great.